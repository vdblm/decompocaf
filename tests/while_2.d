int main(){
    while(ReadInteger()){
        Print("hey");
    }
    while(ReadInteger()){
        Print("hoy");
    }
}
