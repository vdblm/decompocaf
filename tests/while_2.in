1
2
0
5
6
7
0
