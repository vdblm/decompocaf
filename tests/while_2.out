hey
hey
hoy
hoy
hoy
